packages/frontend/src/pages/PasswordRecovery.tsx

```diff
@@ -25,14 +20,18 @@ const PasswordRecovery: FC = () => {
 
     return (
         <Page isFullHeight>
-            <FormWrapper>
-                <LogoWrapper>
-                    <Logo src={LightdashLogo} alt="lightdash logo" />
-                </LogoWrapper>
-                <CardWrapper elevation={2}>
+            <Stack w={400} mt="4xl">
```

Comment:
can you add a FIXME comment here so we update the width to a non-hardcoded value later?